tplinkrouterc6u/client/wdr.py

```diff
@@ -0,0 +1,615 @@
+            oCfg['dns_pri'] = cfg[6] or None
+            oCfg['dns_sec'] = cfg[7] or None
+            self.network.dhcp_cfg = oCfg
+        elif section == "dhcpreserve":
+            item : IPv4Reservation = {}   
+            self.ipv4Reserves = []
+            for i in range(0,len(data['script0']), 3):
+                _dev: HostId = self._findHostInLeases(data['script0'][i])
+                item = IPv4Reservation(
+                    data['script0'][i],
+                    data['script0'][i+1],
+                    _dev.host,
+                    bool(int(data['script0'][i+2]))
+                    )
+                self.ipv4Reserves.append(item)
+        elif section == "dhcplease":
+            self.dhcpLeases = []
+            for i in range(0,len(data['script0']), 4):
+                item = IPv4DHCPLease(
+                    EUI48(data['script0'][i+1]),
+                    IPv4Address(data['script0'][i+2]),
+                    data['script0'][i],
+                    data['script0'][i+3]
+                    )
+                self.dhcpLeases.append(item)
+        elif (section == 'portFwd'):
+            # TODO            
+            pass
+       
+    def _parseSummary(self, raw:str) -> None:
+        data = self._parseRawHTML(raw)
+        tFirm= data['script0'][6]
+        tHard = data['script0'][7]
+        ## WDR3600 v1 00000000
+        tModel = tHard.split(" ")
+        self.firmware= Firmware(tHard, tModel[0], tFirm)
+        self.status = Status()
+        self.status.wan_ipv4_uptime = int(data['script0'][8]) 
+        self.status._lan_ipv4_addr = get_ip(data['script1'][1])
+        self.status._lan_macaddr = EUI48(data['script1'][0])
+
+        self.ipv4status._lan_ipv4_ipaddr = get_ip(data['script1'][1])
+        self.ipv4status._lan_macaddr = EUI48(data['script1'][0])
+
+        self.status._wan_macaddr = EUI48(data['script5'][1])
+        self.status._wan_ipv4_addr = get_ip(data['script5'][2])
+        self.status._wan_ipv4_gateway = get_ip(data['script5'][7])
+
+        self.ipv4status._wan_macaddr = EUI48(data['script5'][1])
+        self.ipv4status._wan_ipv4_ipaddr = get_ip(data['script5'][2])
+        self.ipv4status._wan_ipv4_netmask = ""
+        self.ipv4status._wan_ipv4_gateway = get_ip(data['script5'][7])
+
+        self.status.guest_2g_enable = None
+        self.status.guest_5g_enable =  None
+        self.status.wifi_2g_enable = bool(int(data['script2'][0]))
+        self.status.wifi_5g_enable =  bool(int(data['script3'][0]))
+
+        self.status.conn_type = 'unknown' 
+        self.status.devices = []
+
+    def _get_conn_type(self,n:int) -> str:
+        wantypeinfo = [6,0,"WanDynamicIpCfgRpm.htm",1,"WanStaticIpCfgRpm.htm",2,"PPPoECfgRpm.htm",5,"BPACfgRpm.htm",6,"L2TPCfgRpm.htm",7,"PPTPCfgRpm.htm",0,0 ];
+        wantype_filtered = wantypeinfo[2*n+1]
+
+        wan_type: list = [
+            'Dynamic IP',
+            'Static IP',
+            'PPPoE/Russia PPPoE',
```

Comment:
Maybe better to return 'PPPoE' instead of 'PPPoE/Russia PPPoE', 'L2TP' instead of 'L2TP/Russia L2TP' and 'PPTP' instead of 'PPTP/Russia PPTP' as this client would be used in different countries?